lsan_suppress.txt

```diff
@@ -0,0 +1,15 @@
+leak:obmalloc.c
```

Comment:
This leak suppression casts a very wide net, including other libraries. That diminishes the usefulness somewhat...(ex: if we use `PyObject_Malloc` LSAN will not catch leaks, as we exclude any leaks coming from `obmalloc.c`) but let's see how it goes...